src/niveristand/clientapi/realtimesequencedefinition.py

```diff
@@ -1,192 +1,145 @@
-import os

```

Comment:
What changed here? Is it whitespace?

Reply:
i didn't change anything. maybe it was git changing EOL on commit.